#! /bin/sh /usr/share/dpatch/dpatch-run
## 203_ignore_wrong_versions.dpatch by <andrew.stormont@nexenta.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch taken from StormOS to fixes Nexenta Bug #5727
## DP: Upstream status: not submitted yet


@DPATCH@
--- ./bfd/elflink.c.bak	2011-07-08 19:07:51.733928862 +0100
+++ ./bfd/elflink.c	2011-07-08 19:09:22.145389928 +0100
@@ -4044,12 +4044,7 @@
 
 		  if (verstr == NULL)
 		    {
-		      (*_bfd_error_handler)
-			(_("%B: %s: invalid version %u (max %d)"),
-			 abfd, name, vernum,
-			 elf_tdata (abfd)->cverdefs);
-		      bfd_set_error (bfd_error_bad_value);
-		      goto error_free_vers;
+		      continue;
 		    }
 		}
 	      else
